[DOCS] Replaces outdated integration guides with redirects to versioned documentation (#8173)
great-expectations/great_expectations: docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_meltano.md

sidebar_label: "Meltano"
sidebar_custom_props: { icon: 'img/integrations/meltano_icon.png' }
---

> Meltano is an Open Source DataOps OS that's used to install and configure data applications (Great Expectations, Singer, dbt, Airflow, etc.) that your team's data platform is built on top of, all in one central repository.

Reviewer: ```suggestion
Meltano is an Open Source DataOps OS that you can use to install and configure data applications such as Great Expectations (GX) Singer, dbt, and Airflow with your existing data platform in a central repository.
```
![image](https://github.com/great-expectations/great_expectations/assets/104205257/e038be2d-7865-4a7c-8fe8-c4245589acf3)

> Using Meltano enables teams to easily implement DataOps best practices like configuration as code, code reviews, isolated test environments, CI/CD, etc.  A common use case is to manage ELT pipelines with Meltano and as part of ensuring the quality of the data in those pipelines, teams bring in Great Expectations.

Reviewer: ```suggestion
Meltano can help you implement DataOps best practices such as configuration as code, code reviews, isolated test environments, and CI/CD.  Typically, Metano is used to manage ELT pipelines and to help ensure data quality in pipelines.
```
